R/nba_scraper.R

```diff
@@ -1,26 +1,154 @@
 #' Scrape NBA statistics data from ESPN into CSV file
 #'
-#' Scrape the tabular data from ESPN NBA website into a csv file.
+#' Scrape the tabular data from ESPN NBA website into a csv file using RSelenium.
 #'
-#' @param season_year string
+#' @param season_year int from 2001 to 2019 (upper limit based on latest year)
 #' @param season_type string
 #' @param csv_path string
+#' @param port int with L suffix
+#' @param sel_browser string
 #' @param nba_data_env_name string
```

Comment:
The function should return the dataframe, this parameter should be changed

Reply:
this has been changed accordingly with a more recent commit!